### PR TITLE
feat: Description tooltip on hover

### LINKED ISSUE #18: Description Text Hover
As a user I want to see the whole description text when hovering the description.

ACs:

- [ ] the hover should only appear when there is more text then in two rows displayed
- [ ] the whole description text will be displayed as a hover hint when hovering it
- [ ] use similar hover behaviour like in other OpenSCD hovers

### PR DESCRIPTION
**Pull request by rdsov**
Resolves #18

### CODE REVIEW

**Comment by rdsov** [comment]
Tooltip

File: src/plugin-store-item.svelte [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+$: menuStates = filteredPlugins.map(() => false);
+
+function openPluginMenu(index: number) {
+	menuStates = menuStates.map(() => false);
+	menuStates[index] = true;
+}
+
+function alternateRowColors(index: number) {
+	return `background: rgba(0, 0, 0, ${index % 2 === 0 ? "0" : "0.03"});`;
+}
```
Comment (trusz) [inline src/plugin-store-item.svelte:104]: We usually can solve this with CSS. Is there reason why we do this with JavaScript?

File: src/plugin-store-item.svelte [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+        <List>
+        <Item
+        on:SMUI:action={() =>
+        uninstallExternalPlugin(
+        plugin
+        )}
+        >
+        <Text>Uninstall</Text>
+        </Item>
+        </List>
```
Comment (trusz) [inline src/plugin-store-item.svelte:170]: The indentation seems to be off here

**Review by trusz — APPROVED** [review]

File: src/plugin-store-item.svelte (hunk 1 above)
Comment (rdsov) [reply src/plugin-store-item.svelte:104]: That was just something I was familiar with, I can move it to CSS though

**Pull request merged**